packages/base/spec.gts

```diff
@@ -43,7 +43,7 @@ import Brain from '@cardstack/boxel-icons/brain';
 import { use, resource } from 'ember-resources';
 import { TrackedObject } from 'tracked-built-ins';
 
-export type SpecType = 'card' | 'field' | 'component' | 'app' | 'skill';
+export type SpecType = 'card' | 'field' | 'component' | 'app';
```

Comment:
Ths idea of  a skill as a spec; doesn't really make so much sense. Spec is truer to code than instance. We already handle mangling of instances inside of a listing. This makes it clearer that a spec only pertains to code-related things -- it is unusual that a Skill which reference a single Skill base is indeed a spec; We will only ever have one spec with one specType ='Skill'. 

So after talking to Chris we agreed to remove this